server/app/mocap_server.py

```diff
@@ -352,6 +354,15 @@ def process(self):
                 if os.path.exists(path + self.subjectName + '.bin'):
                     self.index.uploadFile(
                         self.pytorchResults, path + self.subjectName + '.bin')
+
+                # 5.1.3. Upload the _errors.json file.
+                if os.path.exists(path + '_errors.json'):
+                    self.index.uploadFile(
+                        self.errorsFile, path + '_errors.json')
+                else:
+                    print('WARNING! FILE NOT UPLOADED BECAUSE FILE NOT FOUND! ' +
```

Comment:
I think if we don't need this warning, since it's not a problem if this file isn't found. Otherwise LGTM!